tests/torch/fx/test_fast_bias_correction.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) 2024 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List
+
+import pytest
+import torch
+from torch._export import capture_pre_autograd_graph
+
+from nncf.common.factory import NNCFGraphFactory
+from nncf.experimental.torch.fx.transformations import apply_quantization_transformations
+from nncf.quantization.algorithms.fast_bias_correction.torch_fx_backend import FXFastBiasCorrectionAlgoBackend
+from nncf.torch.dynamic_graph.patch_pytorch import disable_patching
+from nncf.torch.model_graph_manager import OPERATORS_WITH_BIAS_METATYPES
+from tests.post_training.test_templates.test_fast_bias_correction import TemplateTestFBCAlgorithm
+
+
+class TestTorchFXFBCAlgorithm(TemplateTestFBCAlgorithm):
+    @staticmethod
+    def list_to_backend_type(data: List) -> torch.Tensor:
+        return torch.Tensor(data)
+
+    @staticmethod
+    def get_backend() -> FXFastBiasCorrectionAlgoBackend:
+        return FXFastBiasCorrectionAlgoBackend
+
+    def _get_fx_model(model: torch.nn.Module):
+        device = next(model.named_parameters())[1].device
+        input_shape = model.INPUT_SIZE
+        if input_shape is None:
+            input_shape = [1, 3, 32, 32]
+        ex_input = torch.ones(input_shape).to(device)
+        model.eval()
+        with disable_patching():
+            exported_model = capture_pre_autograd_graph(model, args=(ex_input,))
+        apply_quantization_transformations(exported_model)
+        return exported_model
+
+    @staticmethod
+    def backend_specific_model(model: torch.nn.Module, tmp_dir: str):
+        fx_model = TestTorchFXFBCAlgorithm._get_fx_model(model)
```

Comment:
Please move code from `_get_fx_model` to `backend_specific_model`

Reply:
Done